serde/src/java/org/apache/hadoop/hive/serde2/OpenCSVSerde.java

```diff
@@ -92,6 +95,7 @@ public void initialize(Configuration configuration, Properties tableProperties,
     separatorChar = getProperty(properties, SEPARATORCHAR, CSVWriter.DEFAULT_SEPARATOR);
     quoteChar = getProperty(properties, QUOTECHAR, CSVWriter.DEFAULT_QUOTE_CHARACTER);
     escapeChar = getProperty(properties, ESCAPECHAR, CSVWriter.DEFAULT_ESCAPE_CHARACTER);
+    applyQuotesToAll = Boolean.parseBoolean(properties.getProperty(APPLYQUOTESTOALL))
```

Comment:
@gigem,  ';' expected

Reply:
Sorry about that.  I'm too old and should know bettter than to at least do a test compile on any changes.  It's fixed now.